src/test/utils/meta_table_manager_test.cpp

```diff
@@ -49,11 +49,29 @@ TEST_F(MetaTableManagerTest, TableBasedMetaData) {
     }
   }
 
+  // Test meta table for chunk sort orders
```

Comment:
Why isn't this part of TableBasedMetaData?

Reply:
Hmm? It is part of `TableBasedMetaData`. Or do you mean the first loop?

Reply:
Yes, I mean the magic thingy that does all sorts of tests where you added `segments_accurate`.

Reply:
The executed encoding and inserts of tuples would not have an effect. I could add the table there but the TBL would not change for the updated version.
Your call.

Reply:
Ah, I didn't realize that `chunk_sort_orders` only contains rows for chunks that are actually ordered. Never mind.